common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt

```diff
@@ -160,6 +160,20 @@ data class BrokerData(val packageName : String,
             "debugLTW"
         )
 
+        @JvmStatic
+        val debugIntuneCE = BrokerData(
+            AuthenticationConstants.Broker.INTUNE_APP_PACKAGE_NAME,
+            AuthenticationConstants.Broker.INTUNE_APP_SHA512_DEBUG_SIGNATURE,
+            "debugIntuneCE"
+        )
+
+        @JvmStatic
+        val prodIntuneCE = BrokerData(
+            AuthenticationConstants.Broker.INTUNE_APP_PACKAGE_NAME,
+            AuthenticationConstants.Broker.INTUNE_APP_SHA512_RELEASE_SIGNATURE,
+            "prodIntuneCE"
+        )
```

Comment:
This seems special case here.. that this is app is not a broker itself but is broker API client. Add a comment here?

Reply:
comment added in the place where this is used.